### PR TITLE
improve speed !

### PR DESCRIPTION
**Pull request by ipatate**

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -3,12 +3,12 @@ lockfileVersion: 5.4
 specifiers:
   '@headlessui/vue': ^1.7.3
   '@nathanchase/nuxt-dayjs-module': ^1.0.11
-  '@nuxt/content': ^2.2.0
```
Comment (xlanex6) [inline pnpm-lock.yaml:6]: Shall we put `pnpm-lock` in `.gitignore` ?

File: cache/db/storage.json [hunk 2]
```diff
@@ -0,0 +1,206 @@
+{
+    "https://chtbl.com/track/79E812/double-slash.ams3.cdn.digitaloceanspaces.com/DS_000_prologue.mp3": {
```
Comment (xlanex6) [inline cache/db/storage.json:2]: do we have to keep ALL url ? 
slug could be enough not ?

Comment (xlanex6) [reply cache/db/storage.json:2]: DB is auto save on new episode ?

**Review by xlanex6 — APPROVED** [review]

File: pnpm-lock.yaml (hunk 1 above)
Comment (ipatate) [reply pnpm-lock.yaml:6]: No, it's better to share this for cache

**Pull request merged**